pkg/kubelet/certificate/transport_test.go

```diff
@@ -209,16 +209,17 @@ func TestRotateShutsDownConnections(t *testing.T) {
 	// its connections to the server.
 	m.setCurrent(client2CertData.certificate)
 
-	err = wait.PollImmediate(time.Millisecond*50, wait.ForeverTestTimeout, func() (done bool, err error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), wait.ForeverTestTimeout)
+	defer cancel()
+	if err := wait.PollUntilContextCancel(ctx, time.Millisecond*50, true, func(_ context.Context) (done bool, err error) {
```

Comment:
Can't we pass this `context`? Then we can remove the `context.TODO()` in the function.

Reply:
Good question, I used the `context.TODO()ˋ because it was already used a few lines above. I will check.